alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureWorkflowRestApiIT.kt

```diff
@@ -63,7 +63,7 @@ import org.opensearch.test.junit.annotations.TestLogging
 import java.time.Instant
 
 // TODO investigate flaky nature of tests. not reproducible in local but fails in jenkins CI
-@Ignore
+@AwaitsFix(bugUrl = "Replace with issue URL")
```

Comment:
I created this issue to track the investigation of these flaky tests. Could you revise this PR to include the issue link as the `bugUrl`?
https://github.com/opensearch-project/alerting/issues/1246

Reply:
Sure! Just did that. @AWSHurneyt